FRA_WE_BACK/core/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from django.http 	        import JsonResponse    
+
+from FRA_WE_BACK.settings     import SECRET_KEY                             
+from users.models             import User
+
+
+def LogInDecorator(func):
```

Comment:
함수는 클래스와 네이밍 컨벤션이 다릅니다. 확인해주세요
```suggestion
def log_in_decorator(func):
```